res/values/arrays.xml

```diff
@@ -42,6 +43,7 @@
     <item>QWERTY (Norwegian)</item>
     <item>QWERTY (Swedish)</item>
     <item>QWERTY (Hungarian)</item>
+    <item>QWERTY (Hebrew)</item>
```

Comment:
Is it commonly called like that ? It doesn't seem like qwerty at all, with punctuation and backspace on the first row and 11 columns.

Reply:
Wikipedia calls it "Standard Hebrew keyboard": https://en.wikipedia.org/wiki/Hebrew_keyboard
I think you can rename the layout to "Hebrew" here.

Reply:
Physical Hebrew keyboard layouts have QWERTY capitals built-into the shifting / Caps Lock but Google didn't implement it (which is why I moved the shift to a gesture) so you make a valid point.

I guess I'll try and see how much I can implement out of SI-1452-1 first and adjust the naming accordingly.

Ref:
SI-1452-1: http://standards.har-el.com/Projects/kb/PRD_01452-01-00-0-20.pdf (p.10 for the layout picture)
SI-1452-2: http://standards.har-el.com/Projects/kb/Draft%20SI%2001452-02-00-0%20.pdf (p.11 for the layout picture)

Note the latter standard wasn't widely adopted.

Reply:
Please add the layout that you would use.

The name "QWERTY (Hebrew)" seems closer to SI-1452-1 and 2, which is not the layout you wrote. I'm not sure such a layout is useful when we have the "secondary" layout option to switch between Hebrew and QWERTY easily, except if it's the layout that you wish use.